javascript/javascript1/week3/phone_usage.js

```diff
@@ -0,0 +1,102 @@
+//phone usage
+const activities = [];
+const USER_LIMIT = 150;
+const WARNING_THRESHOLD = USER_LIMIT * 0.8;
+const categories = {
+  Social: ["Instagram", "Tik Tok", "Messenger", "WhatsApp"],
+  Intertainment: ["Netflix", "Games", "YouTube"],
+  Health: ["Fitbit"],
+};
+let totalDuration = 0;
+
+function addActivity(activity, duration) {
```

Comment:
The task was suggesting adding date together with the `activity` and `duration` parameters. eg.
```
addActivity("23/7-18", "Youtube", 30);
```
This would require a lot of code change, and I know it can be frustrating. You've taken a different approach, and have added a lot extra logic and code, so I'd leave it up to you to decide if you want to experiment and fiddle more (: